### PR TITLE
MM5-5273 Add import/export labels

### PR DESCRIPTION
**Pull request by bbedigi**

### CODE REVIEW

File: MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl [hunk 1]
```diff
@@ -1054,6 +1054,408 @@ resource configservice_label portal_config_manager_portal_labels_language {
...
+  key = 'PORTAL_CONFIG_MANAGER_PORTAL_LABELS_EXPORT_NOTIFICATION_ERROR_BODY'
+  group = 'Config Manager'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Error exporting labels. Check your browser\'s console for more details.'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Fejl ved eksport af mærkaterne. Tjek din browsers konsol for flere detaljer.'
```
Comment (Poiar) [inline MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl:1260]:
```suggestion
      default_translation = 'Fejl under label-eksport. Tjek din browsers konsol for flere detaljer.'
```

File: MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl [hunk 2]
```diff
@@ -1054,6 +1054,408 @@ resource configservice_label portal_config_manager_portal_labels_language {
...
+  key = 'PORTAL_CONFIG_MANAGER_PORTAL_LABELS_IMPORT_DIALOG_TITLE'
+  group = 'Config Manager'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'WARNING: Overwriting labels'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'ADVARSEL: Overskrivning af mærkater'
```
Comment (Poiar) [inline MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl:1276]:
```suggestion
      default_translation = 'ADVARSEL: Overskrivning af labels'
```

File: MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl [hunk 3]
```diff
@@ -1054,6 +1054,408 @@ resource configservice_label portal_config_manager_portal_labels_language {
...
+  key = 'PORTAL_CONFIG_MANAGER_PORTAL_LABELS_IMPORT_DIALOG_BODY'
+  group = 'Config Manager'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Are you sure you want to overwrite all \'{{language}}\' labels with \'{{filename}}\'? \n\n We recommend backing up labels before overwriting!'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Er du sikker på, at du vil overskrive alle \'{{language}}\' etiketter med \'{{filename}}\'? \n\n Vi anbefaler at sikkerhedskopiere etiketter før overskrivning!'
```
Comment (Poiar) [inline MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl:1293]:
```suggestion
      default_translation = 'Er du sikker på, at du vil overskrive alle \'{{language}}\'e labels med \'{{filename}}\'? \n\n Vi anbefaler at sikkerhedskopierer labels før overskrivning!'
```

File: MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl [hunk 4]
```diff
@@ -1054,6 +1054,408 @@ resource configservice_label portal_config_manager_portal_labels_language {
...
+  key = 'PORTAL_CONFIG_MANAGER_PORTAL_LABELS_IMPORT_NOTIFICATION_SUCCESS_BODY'
+  group = 'Config Manager'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'The labels have been imported. Refresh your page to see any changes.'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Mærkaterne er blevet downloadet til din browsers downloadmappe'
```
Comment (Poiar) [inline MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl:1357]:
```suggestion
      default_translation = 'Labels er blevet importeret. Opdater siden for at se ændringerne.'
```

File: MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl [hunk 5]
```diff
@@ -1054,6 +1054,408 @@ resource configservice_label portal_config_manager_portal_labels_language {
...
+  ]
+}
+
+resource configservice_label portal_config_manager_portal_labels_import_notification_success_body {
+  key = 'PORTAL_CONFIG_MANAGER_PORTAL_LABELS_IMPORT_NOTIFICATION_SUCCESS_BODY'
+  group = 'Config Manager'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'The labels have been imported. Refresh your page to see any changes.'
```
Comment (Poiar) [inline MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl:1353]:
```suggestion
      default_translation = 'The labels have been imported. Refresh the page to see changes.'
```

File: MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl [hunk 6]
```diff
@@ -1054,6 +1054,408 @@ resource configservice_label portal_config_manager_portal_labels_language {
...
+  key = 'PORTAL_CONFIG_MANAGER_PORTAL_LABELS_IMPORT_NOTIFICATION_ERROR_BODY'
+  group = 'Config Manager'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Error importing labels. Check your browser\'s console for more details.'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Fejl ved import af mærkaterne. Tjek din browsers konsol for flere detaljer.'
```
Comment (Poiar) [inline MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl:1422]:
```suggestion
      default_translation = 'Fejl under label-import. Tjek din browsers konsol for flere detaljer.'
```

**Review by Poiar — CHANGES_REQUESTED** [review]

File: MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl [hunk 7]
```diff
@@ -1054,6 +1054,408 @@ resource configservice_label portal_config_manager_portal_labels_language {
...
+  key = 'PORTAL_CONFIG_MANAGER_PORTAL_LABELS_DESCRIPTION_IMPORT_EXPORT'
+  group = 'Config Manager'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Export the labels in your chosen language as an Excel sheet using the Export button. Any changes you make in the Excel file can be imported using the Import button.'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Eksporter mærkater på dit valgte sprog som et Excel-ark ved at bruge knappen Eksport. Eventuelle ændringer, du foretager i Excel-filen, kan importeres ved hjælp af knappen Import.'
```
[the remark this answers is not in the document]
Comment (Poiar) [reply MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl:1083]:
```suggestion
      default_translation = 'Eksporter alle {{sprog}}e labels til et Excel-ark ved at trykke Eksporter. Ændringer du foretager i Excel-arket, kan importeres ved at trykke Importer.'
```

File: MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl [hunk 8]
```diff
@@ -1054,6 +1054,408 @@ resource configservice_label portal_config_manager_portal_labels_language {
...
+  ]
+}
+
+resource configservice_label portal_config_manager_portal_labels_description_import_export {
+  key = 'PORTAL_CONFIG_MANAGER_PORTAL_LABELS_DESCRIPTION_IMPORT_EXPORT'
+  group = 'Config Manager'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Export the labels in your chosen language as an Excel sheet using the Export button. Any changes you make in the Excel file can be imported using the Import button.'
```
Comment (Poiar) [inline MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl:1079]:
```suggestion
      default_translation = 'Export all {{language}} labels as an Excel sheet using the Export button. Any changes you make in the Excel file can be imported using the Import button.'
```

File: MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl [hunk 9]
```diff
@@ -1054,6 +1054,408 @@ resource configservice_label portal_config_manager_portal_labels_language {
...
+}
+
+
+resource configservice_label portal_config_manager_portal_labels_export_dialog_body {
+  key = 'PORTAL_CONFIG_MANAGER_PORTAL_LABELS_EXPORT_DIALOG_BODY'
+  group = 'Config Manager'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Are you sure you want to export all labels for \'{{language}}\'?'
```
Comment (Poiar) [inline MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl:1160]:
```suggestion
      default_translation = 'Are you sure you want to export all {{language}} labels?'
```

File: MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl [hunk 10]
```diff
@@ -1054,6 +1054,408 @@ resource configservice_label portal_config_manager_portal_labels_language {
...
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Er du sikker på, at du vil eksportere alle mærkater for \'{{sprog}}\'?'
```
[the remark this answers is not in the document]
Comment (Poiar) [reply MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl:1164]:
```suggestion
      default_translation = 'Er du sikker på, at du vil eksportere alle {{sprog}}e labels?'
```

File: MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl [hunk 11]
```diff
@@ -1054,6 +1054,408 @@ resource configservice_label portal_config_manager_portal_labels_language {
...
+  key = 'PORTAL_CONFIG_MANAGER_PORTAL_LABELS_EXPORT_NOTIFICATION_SUCCESS_BODY'
+  group = 'Config Manager'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'The labels have been downloaded to your browser\'s download directory'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Mærkaterne er blevet downloadet til din browsers downloadmappe'
```
[the remark this answers is not in the document]
Comment (Poiar) [reply MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl:1228]:
```suggestion
      default_translation = 'Alle {{sprog}}e labels er downloadet til din browsers downloadmappe'
```

File: MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl [hunk 12]
```diff
@@ -1054,6 +1054,408 @@ resource configservice_label portal_config_manager_portal_labels_language {
...
+  ]
+}
+
+resource configservice_label portal_config_manager_portal_labels_export_notification_success_body {
+  key = 'PORTAL_CONFIG_MANAGER_PORTAL_LABELS_EXPORT_NOTIFICATION_SUCCESS_BODY'
+  group = 'Config Manager'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'The labels have been downloaded to your browser\'s download directory'
```
Comment (Poiar) [inline MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl:1224]:
```suggestion
      default_translation = 'All {{language}} labels have been downloaded to your browser\'s download directory'
```

File: MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl (hunk 2 above)
Comment (Poiar) [reply MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl:1276]:
```suggestion
      default_translation = 'ADVARSEL: Overskrivning af alle {{sprog}}e labels'
```

File: MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl [hunk 13]
```diff
@@ -1054,6 +1054,408 @@ resource configservice_label portal_config_manager_portal_labels_language {
...
+  ]
+}
+
+resource configservice_label portal_config_manager_portal_labels_import_dialog_title {
+  key = 'PORTAL_CONFIG_MANAGER_PORTAL_LABELS_IMPORT_DIALOG_TITLE'
+  group = 'Config Manager'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'WARNING: Overwriting labels'
```
Comment (Poiar) [inline MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl:1272]:
```suggestion
      default_translation = 'WARNING: Overwriting all {{language}} labels'
```

File: MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl (hunk 8 above)
Comment (bbedigi) [reply MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl:1079]: this is the general description, no language available here

File: MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl (hunk 13 above)
Comment (bbedigi) [reply MM/5.8.0/config/media_manager_5/labels/Config Manager.dcl:1272]: Think that's too long, the language is in the body anyway

**Review by Poiar — APPROVED** [review]

**Pull request merged**